rpc/handlers.go

```diff
@@ -182,34 +182,35 @@ func adaptDeclareTransaction(t *core.DeclareTransaction) *Transaction {
 
 	if t.Version.Equal(new(felt.Felt).SetUint64(2)) {
 		txn.CompiledClassHash = nil // todo: add when we have support for Declare V2
-	} else if !t.Version.IsZero() && !t.Version.IsOne() {
```

Comment:
Was the removal of this specific 'else if' intentional?

Reply:
Yes since we trust the source of information.